ccd-definition/CaseField.json

```diff
@@ -50,9 +50,83 @@
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "PERSONAL_INJURY",
-    "ID": "statementOfTruth",
+    "ID": "claimStatementOfTruth",
     "Label": "Statement of truth",
     "FieldType": "StatementOfTruth",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "PERSONAL_INJURY",
+    "ID": "servedDocuments",
+    "Label": "Which documents did you serve?",
+    "FieldType": "MultiSelectList",
+    "FieldTypeParameter": "ServedDocuments",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "PERSONAL_INJURY",
+    "ID": "servedDocumentsOther",
+    "Label": "List them here:",
+    "FieldType": "TextArea",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "PERSONAL_INJURY",
+    "ID": "servedDocumentFiles",
+    "Label": "Upload your documents",
+    "FieldType": "ServedDocumentFiles",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "PERSONAL_INJURY",
+    "ID": "serviceMethod",
+    "Label": "How did you serve the documents?",
+    "FieldType": "FixedRadioList",
+    "FieldTypeParameter": "ServiceMethod",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "PERSONAL_INJURY",
+    "ID": "serviceLocation",
+    "Label": "Where did you serve the documents?",
+    "FieldType": "ServiceLocation",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "PERSONAL_INJURY",
+    "ID": "serviceDate",
+    "Label": "When did you serve the documents?",
+    "FieldType": "Date",
+    "SecurityClassification": "Public"
+  },
```

Comment:
These items named 'service'Something are understandable within the context of the ticket, but when looking purely at the data model, it's not clear what they refer to, particularly when 'service' is already a loaded term. Could it be something more like 'servedDocuments'Something? Happy to discuss.

Reply:
Naming things is one of the hardest parts about being a developer I swear 😅 

umm off the top of my head:
- documentServiceMethod, documentServiceLocation, documentServiceDate...
- methodDocumentsServed etc...
- methodOfService etc...

Reply:
This is why it should be my problem so you can do all the clever stuff :) 

If there is only one type of documents that get served, then documentServiceThingumybob is cool.

Am I correct in understanding that these can't be a complexType because the data capture is split across pages?

Reply:
it's my understanding a defendant is only served once, this might be something we need to take into consideration for 2v1, but as it's not a part of release one I haven't worried about it too much yet.

Correct that these can't be a single complex type because a complex type cannot be split across multiple pages

Reply:
Ok, I'll have a think about how to name these things in a multi-party world.

Thanks for clarifying on the complex type issue!